src/web/aws-cloudwatch/CloudWatch.jsx

```diff
@@ -3,11 +3,13 @@ import PropTypes from 'prop-types';
 import { Col, Row } from 'react-bootstrap';
 
 import Wizard from 'components/common/Wizard';
+import PageHeader from 'components/common/PageHeader';
 
 import StepAuthorize from './StepAuthorize';
 import StepKinesis from './StepKinesis';
 import StepHealthCheck from './StepHealthCheck';
 import StepReview from './StepReview';
+import { DEFAULT_SETTINGS } from './util';
```

Comment:
I think we should rename this to `defaults.js` or `default_settings.js` to make its intention clearer.